core-web/libs/portlets/edit-ema/portlet/src/lib/services/dot-page-api.service.ts

```diff
@@ -74,7 +75,7 @@ export class DotPageApiService {
         // Remove trailing and leading slashes
         const url = params.url.replace(/^\/+|\/+$/g, '');
 
-        const apiUrl = `/api/v1/page/json/${url}?language_id=${params.language_id}&com.dotmarketing.persona.id=${params['com.dotmarketing.persona.id']}`;
+        const apiUrl = `/api/v1/page/render/${url}?language_id=${params.language_id}&com.dotmarketing.persona.id=${params['com.dotmarketing.persona.id']}`;
```

Comment:
`render` is slower that `json` can we do this only for non headless pages?

Reply:
Added!